code/game/machinery/syndicatebomb.dm

```diff
@@ -530,6 +530,48 @@
 
 		qdel(G)
 
+/obj/item/bombcore/toxins
+	name = "toxins payload"
+	desc = "A payload casing designed to secure a gas based bomb. Must be loaded with a tank transfer valve and installed into a plasteel bomb frame in order to be detonated."
+	origin_tech = "materials=1;engineering=1"
+	icon_state = "chemcore"
+	var/obj/item/transfer_valve/ttv = null
+
+/obj/item/bombcore/toxins/attackby(obj/item/I, mob/user)
+	if(iscrowbar(I) && ttv)
+		playsound(loc, I.usesound, 50, 1)
+		ttv.forceMove(get_turf(src))
+		ttv = null
+	else if(istype(I, /obj/item/transfer_valve))
+		if(!ttv && !check_attached(I))
```

Comment:
Why prevent attached assemblies? The bombs can be detonated early anyhow.

Reply:
The bomb core is intended to be a replacement detonator instead of an attachment. You can't attach two remote signallers to one tank transfer valve (if you can, let me know how i'd actually like to know), it seems logical to me you cannot have both the core acting as detonator and an attached signaller/timer also acting as detonator.

Reply:
I don't really see any usability upsides to making the valve part of crafting, whether in game or in the code... This is also consistent with how the chemcore can have beakers added and removed.

Reply:
An empty valve. Then have the tanks inserted/removed individually on the core.

Reply:
I get what you mean, I just don't see the point. it'd require a way to deconstruct the core aswell if you wanted the TTV back if you changed your mind/if someone already made the TTV into toxincores just cuz, and would also probably require extra spritework for the sake of clarity.